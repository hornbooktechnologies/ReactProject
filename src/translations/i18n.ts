import { initReactI18next } from 'react-i18next'
import i18n from 'i18next'
import LanguageDetector from 'i18next-browser-languagedetector'
import { TRANSLATIONS_EN } from './en/Translations'
import { TRANSLATIONS_JA } from './ja/Translations'

i18n
  .use(LanguageDetector)
  .use(initReactI18next)
  .init({
    lng: 'en',
    resources: {
      en: {
        translation: TRANSLATIONS_EN,
      },
      ja: {
        translation: TRANSLATIONS_JA,
      },
    },
  })

export default i18n
