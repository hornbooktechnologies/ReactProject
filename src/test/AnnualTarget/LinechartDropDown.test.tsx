import React from "react";
import LinechartDropDown from "../../components/common/Dropdown";
import renderer from "react-test-renderer";
import { I18nextProvider } from "react-i18next";
import i18n from "../../i18nForTests";
describe("Component: LinechartDropDown", () => {
  it("LinechartDropDown renders correctly", () => {
    const tree = renderer
      .create(
        <I18nextProvider i18n={i18n}>
          <LinechartDropDown />
        </I18nextProvider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
