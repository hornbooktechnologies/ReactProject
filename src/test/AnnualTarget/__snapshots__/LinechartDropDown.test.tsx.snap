// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: LinechartDropDown LinechartDropDown renders correctly 1`] = `
<div
  className="form-control"
>
  <label
    className="uk-custom-label"
  >
    <span>
      LABELS.DISPLAY
    </span>
  </label>
  <div
    className=" css-b62m3t-container"
    onKeyDown={[Function]}
  >
    <span
      className="css-1f43avz-a11yText-A11yText"
      id="react-select-2-live-region"
    />
    <span
      aria-atomic="false"
      aria-live="polite"
      aria-relevant="additions text"
      className="css-1f43avz-a11yText-A11yText"
    />
    <div
      className="form_select_input__control css-1s2u09g-control"
      onMouseDown={[Function]}
      onTouchEnd={[Function]}
    >
      <div
        className="form_select_input__value-container form_select_input__value-container--has-value css-319lph-ValueContainer"
      >
        <div
          className="form_select_input__single-value css-qc6sy-singleValue"
        >
          DROPDOWN_OPTIONS.ANNUAL_VIEW
        </div>
        <input
          aria-autocomplete="list"
          aria-controls="react-select-2-listbox"
          aria-expanded={false}
          aria-haspopup={true}
          aria-owns="react-select-2-listbox"
          aria-readonly={true}
          className="css-mohuvp-dummyInput-DummyInput"
          disabled={false}
          id="react-select-2-input"
          inputMode="none"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          role="combobox"
          tabIndex={0}
          value=""
        />
      </div>
      <div
        className="form_select_input__indicators css-1hb7zxy-IndicatorsContainer"
      >
        <span
          className="form_select_input__indicator-separator css-1okebmr-indicatorSeparator"
        />
        <div
          aria-hidden="true"
          className="form_select_input__indicator form_select_input__dropdown-indicator css-tlfecz-indicatorContainer"
          onMouseDown={[Function]}
          onTouchEnd={[Function]}
        >
          <svg
            aria-hidden="true"
            className="css-tj5bde-Svg"
            focusable="false"
            height={20}
            viewBox="0 0 20 20"
            width={20}
          >
            <path
              d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
            />
          </svg>
        </div>
      </div>
    </div>
  </div>
</div>
`;
